api-reference/fees/fee-object.mdx

```diff
@@ -123,45 +191,158 @@ description: "This object represents a line item of an invoice, handling amount,
+            </Expandable>    
+          </ResponseField>
+        ### Case of a fee linked to a package charge model
+        <ResponseField name="free_units" type="string">
+          The quantity of units that are provided free of charge for each billing period in a `package` charge model.
+        </ResponseField>
+        <ResponseField name="paid_units" type="string">
+          The quantity of units that are not provided free of charge for each billing period in a `package` charge model.
+        </ResponseField>
+        <ResponseField name="per_package_size" type="integer">
+          The quantity of units included, defined for `package` or `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="per_package_unit_amount" type="string">
+          Total amount to charge for received `paid_units``, defined for `package` or `percentage` charge model.
+        </ResponseField>
+        ### Case of a fee linked to a percentage charge model
+        <ResponseField name="units" type="string">
+          The total units received in Lago for the `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="free_events" type="integer">
+          Total number of free events allowed for the `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="free_units" type="string">
+          The quantity of units that are provided free of charge for each billing period in a `package` or `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="paid_units" type="string">
+          The quantity of units that are not provided free of charge for each billing period in a `package` or `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="per_unit_amount" type="string">
+          Amount per unit for the `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="per_unit_total_amount" type="string">
+          Total amount of received units to be charged for the `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="paid_events" type="integer">
+          Total number of paid events for the `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="fixed_fee_unit_amount" type="string">
+          Fixed fee unit price per received paid_event for the `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="fixed_fee_total_amount" type="string">
+          Total amount to charge for received paid_events for the `percentage` charge model.
+        </ResponseField>
+        <ResponseField name="min_max_adjustment_total_amount" type="string">
+          Total adjustment amount linked to minimum and maximum spending per transaction for the `percentage` charge model.
+        </ResponseField>
+        ### Case of a fee linked to a standard charge model
+        <ResponseField name="fee.unit_precise_amount" type="string">
```

Comment:
it's `precise_unit_amount`, not only for standard charge model, and on fee object, not on `amount_details`

Reply:
I've edited this section and removed the `Standard charge model` section, it should not be in the `amount_details` you're right 🙌